src/config.rkt

```diff
@@ -131,8 +132,23 @@
 
 (define resetters '())
 
-(define (register-reset fn #:priority [priority 0])
+(define (register-reset! fn #:priority [priority 0])
```

Comment:
Do you remember why we changed this? Is the new define-resetter API better? I assume it has something to do with threads?

Reply:
Each thread now has it's own set of threads. Wasn't a big deal before, but Odyssey causes many threads to interact with Herbie's caches at the same time.